src/app/home-page/home-page.module.ts

```diff
@@ -1,14 +1,18 @@
 import { NgModule } from '@angular/core';
 import { CommonModule } from '@angular/common';
+import { SharedModule } from '../shared/shared.module';
 
 import { HomePageRoutingModule } from './home-page-routing.module';
 import { HomePageComponent } from './components/home-page';
+import { JoinUsComponent } from './components/join-us/join-us.component';
```

Comment:
@NFalinska Honey, it should be shared component as we discussed multiple times.

1. Please move to shared components folder.
2 add into shared declaration and exports
3 import to HomePage only SharedModule